.github/workflows/tests.yml

```diff
@@ -306,10 +306,82 @@ jobs:
       - run: .ci/scripts/test_synapse_port_db.sh
 
   complement:
-    if: ${{ !failure() && !cancelled() }}
+    if: "${{ !failure() && !cancelled() }}"
     needs: linting-done
     runs-on: ubuntu-latest
 
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # GHA requires all matrix configurations to have at least one value. We don't want to set one here though.
+          - _: monolith
```

Comment:
this is a matrix with one job?

Reply:
I was intending to leave it as a placeholder. This way, the job name also includes the correct suffix and it will be easy to make good on getting rid of the duplicate job in the future. (Ideally I was hoping to just copy the rest of the job with a YAML anchor, but that didn't pan out because GitHub Flavoured YAML doesn't support it)